Source/ACE.AuthApi.Host/ACE.AuthApi.Host.csproj

```diff
@@ -27,14 +27,14 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
+    <OutputPath>..\ACE\bin\$(Platform)\$(Configuration)\</OutputPath>
```

Comment:
we need to remove the "AnyCPU" bindings.  i missed them, apparently, but just FYI.